rename-css.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -e
```

Comment:
Does this apply to all the subshells you create. In general, `set -e` is a bad idea.